docs/manual/cluster-admin/how-to-add-and-remove-nodes.md

```diff
@@ -2,154 +2,77 @@
-        origin1:
-        origin2:
-        origin3:
-    k8s-cluster:
-      children:
-        kube-node:
-        kube-master:
-    calico-rr:
-      hosts: {}
-``` 
-
-Go into folder `~/pai-deploy/kubespray/`, run:
-
-```bash
-ansible-playbook -i inventory/pai/hosts.yml cluster.yml -b --become-user=root --limit=new-worker-node-0,new-worker-node-1 -e "@inventory/pai/openpai.yml"
-```
-
-The nodes to add are specified with the `--limit` flag.
-
-### Update OpenPAI Service Configuration
-
-Find your [service configuration file `layout.yaml` and `services-configuration.yaml`](./basic-management-operations.md#pai-service-management-and-paictl) in  `~/pai-deploy/cluster-cfg`.
-
-- Add the new node into `machine-list` field in `layout.yaml`, create a new `machine-sku` if necessary. Refer to [layout.yaml](./installation-guide.md#layoutyaml-format) for schema requirements.
-
-```yaml
-machine-list:
-  - hostname: new-worker-node--0
-    hostip: x.x.x.x
-    machine-type: xxx-sku
-    pai-worker: "true"
-
-  - hostname: new-worker-node-1
-    hostip: x.x.x.x
-    machine-type: xxx-sku
-    pai-worker: "true"
-```
-
-- If you are using hived scheduler, you should modify its setting in `services-configuration.yaml` properly. Please refer to [how to set up virtual clusters](./how-to-set-up-virtual-clusters.md) and the [hived scheduler doc](https://github.com/microsoft/hivedscheduler/blob/master/doc/user-manual.md) for details. If you are using Kubernetes default scheduler, you can skip this step.
-
-- Stop the service, push the latest configuration, and then start related services:
-
-```bash
-./paictl.py service stop -n cluster-configuration hivedscheduler rest-server job-exporter
-./paictl.py config push -p <config-folder> -m service
-./paictl.py service start -n cluster-configuration hivedscheduler rest-server job-exporter
-```
-
-If you have configured any PV/PVC storage, please confirm the added worker node meets the PV's requirements. See [Confirm Worker Nodes Environment](./how-to-set-up-storage.md#confirm-environment-on-worker-nodes) for details.
-
-## How to Remove Nodes
-
-Please refer to the operation of adding nodes. They are very similar.
-
-To remove nodes from the cluster, there is no need to modify `hosts.yml`. 
-Go into `~/pai-deploy/kubespray/`, run
-
-```bash
-ansible-playbook -i inventory/pai/hosts.yml remove-node.yml -b --become-user=root -e "node=worker-node-to-remove-0,worker-node-to-remove-1" -e "@inventory/pai/openpai.yml"
-``` 
-
-The nodes to remove are specified with the `-e` flag.
-
-Modify the `layout.yaml` and `services-configuration.yaml`.
-
-Stop the service, push the latest configuration, and then start related services:
-
-```bash
-./paictl.py service stop -n cluster-configuration hivedscheduler rest-server job-exporter
-./paictl.py config push -p <config-folder> -m service
-./paictl.py service start -n cluster-configuration hivedscheduler rest-server job-exporter
-```
+## Preparation
+
+### For Adding Nodes
```

Comment:
The title is not clear, it's better to summarize the actions to take in this subsection